presto-main/src/main/java/io/prestosql/operator/scalar/JoniRegexpFunctions.java

```diff
@@ -306,4 +306,74 @@ private static void validateGroup(long group, Region region)
             throw new PrestoException(INVALID_FUNCTION_ARGUMENT, format("Pattern has %d groups. Cannot access group %d", region.numRegs - 1, group));
         }
     }
+
+    @ScalarFunction
+    @Description("returns the beginning position of the matched substring")
+    @LiteralParameters("x")
+    @SqlType(StandardTypes.INTEGER)
+    public static long regexpPosition(@SqlType("varchar(x)") Slice source, @SqlType(JoniRegexpType.NAME) Regex pattern)
```

Comment:
Please replace `Regex` with `JoniRegexp`. Otherwise, it will throw "Unsupported type parameters" even if we pass   the right pattern (e.g. `REGEXP_POSITION('a,b,c', 'a')`)

Reply:
Hi,  
There's no class named "JoniRegexp". It's fine to use class "regex" cause it belongs to packge "io.airlift.joni". The unit test of this function passed.

Reply:
`JoniRegexp` exactly exist. Please see the other methods in the same class. Also, could you test again? My test example throws an exception in my environment.

Reply:
Thanks,  I found it! This class was changed two days ago, my local file is out of date.